executions/graphql-kotlin-transaction-batcher/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+description = "Transaction Batcher"
+
+val reactiveStreamsVersion: String by project
+val junitVersion: String by project
+val slf4jVersion: String by project
+val mockkVersion: String by project
+val reactorVersion: String by project
+val reactorExtensionsVersion: String by project
+
+dependencies {
+    implementation("org.reactivestreams:reactive-streams:$reactiveStreamsVersion")
+    implementation("org.slf4j:slf4j-api:$slf4jVersion")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:$junitVersion")
```

Comment:
just a nit but can you make the deps within `testImplementation` sorted in alphanumerical order?